src/Microsoft.Diagnostics.EventFlow.Inputs.MicrosoftLogging/Logger/EventFlowLogger.cs

```diff
@@ -47,6 +48,8 @@ public void Log<TState>(Extensions.Logging.LogLevel logLevel, EventId eventId, T
             }
 
             var scope = EventFlowLoggerScope.Current;
+            List<string> scopePropertyValues = new List<string>(5);
```

Comment:
EventFlow today is not very good at avoiding allocations, but let's try to make things as lean as necessary. So

1. Let's use a pool of lists (or stacks, if you follow my other suggestion) and take a stack from the pool, or create it as necessary. You can use `ConcurrentBag<T>` for that, see https://github.com/Azure/diagnostics-eventflow/blob/master/src/Microsoft.Diagnostics.EventFlow.Core/Implementations/DiagnosticPipeline.cs#L268 for example how to do that.
2. Avoid using a scope value stack at all if there is no scope.